core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -284,6 +284,13 @@ class CityInfoReligionManager {
         }
         return addedPressure
     }
+
+    fun isProtectedByInquisitor(): Boolean {
+        for (tile in cityInfo.getCenterTile().neighbors)
```

Comment:
Should this not also test the center square, as inquisituers can also be there?

Reply:
I suppose so.

Reply:
fixed